library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +68,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    private Map<String,DayViewDecorator> dayViewDecorators;
```

Comment:
We'd also need to add a way to invalidate decorators if they dynamically update one.